fortuna/src/marketplace/Marketplace.js

```diff
@@ -7,45 +7,45 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 // Marketplace component.
 class Marketplace extends React.Component<{||}> {
 
-  render(){
-    return (
-      <div id = "Parent">
-        <div class = "row">
-          <div class = "col-md-4">
-	    <Link to="/MainMenu">
-	      <button type="button" class="btn btn-secondary btn-lg">&lt;- Back to Main Menu</button>
-	    </Link>
-	  </div>
-          <h1 class = "col-md-4 text-center">Marketplace</h1>
-          <h4 class = "col-md-4 text-right">Currency: $9432</h4>
-        </div>
-        <div class = "row mt-5">
-          <div class = "col-md-4">
-            <h2>Available Tanks</h2>
-            <h6>Tank One</h6>
-            <h6>Tank Two</h6>
-            <h6>Tank Three</h6>
-            <h6>Tank Four</h6>
-            <h6>Tank Five</h6>
-            <h6>Tank Six</h6>
-            <h6>Tank Seven</h6>
-            <h6>Tank Eight</h6>
-          </div>
-          <div class = "col-md-4">
-            <h2 class = "text-center">Selected Tank</h2>
-          </div>
-          <div class = "col-md-4">
-            <h2 class = "text-right">Components</h2>
-            <h6 class = "text-right">Tank One Cannon</h6>
-            <h6 class = "text-right">Tank One Tracks</h6>
-            <h6 class = "text-right">Tank One Scanner</h6>
-            <h6 class = "text-right">Tank One Jammer</h6>
-            <h6 class = "text-right">Tank One Base</h6>
-          </div>
-        </div>
-      </div>
-    );
-  }
+  	render(){
+		return (
+	  		<div id="Parent">
+				<div class="row">
```

Comment:
These should be className, not class. I know it isn't your fault, but I must have missed it on the previous review. If you want to change them all, that would be great. If not it's not a big deal either way.